cfgov/unprocessed/js/modules/util/arrays.js

```diff
@@ -0,0 +1,28 @@
+function forEach( array, callback, scope ) {
```

Comment:
This isn't replacing the default `forEach` but rather creating a looping shortcut for when using `querySelectorAll()`

https://css-tricks.com/snippets/javascript/loop-queryselectorall-matches/